### ISSUE TITLE
4.37 Updating RRHStatus should not use RRHHousedDays

### ISSUE THREAD

**Issue by tcalder** [abtassociates/LSASampleCode#24]
This is a bit invovled so bear with me for a second.
https://github.com/MollyMcEvilley/LSASampleCode/blob/b73a71cf526231a029240074e09dce3f1b2bb4cf/LSASampleCode.sql#L2478-L2491
![image](https://user-images.githubusercontent.com/44095973/47316481-7b33b680-d604-11e8-8591-59cb7db070a6.png)
Currently when setting RRHStatus = 2  the code uses RRHHousedDays and RRHPreMoveInDays rather thans sysStatus as outlined in the specs. 

When looking at the source of sysStatus in the specs we see (4.34): 
![image](https://user-images.githubusercontent.com/44095973/47316608-c8178d00-d604-11e8-82df-c71b12172d7f.png)
![image](https://user-images.githubusercontent.com/44095973/47316729-06ad4780-d605-11e8-8e8e-af6ab9d214f7.png)

This seems to imply that when using setting RRHStatus = 2 we should not be including `where RRHHousedDays > 0` as proxy for sysStatus, since sysStatus=6 should only represent those days where you are in an RRHProgram but not housed (sysStatus 2 is for HousedDays).

In conclusion it seems that RRHHousedDays should not be used in this section.

**Comment by MollyMcEvilley** [comment]
This is a good point -- thanks.
